R/sf.R

```diff
@@ -255,6 +255,10 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
       datum = self$datum
     )
 
+	# remove tick labels not on axes 1 (bottom) and 2 (left)
```

Comment:
It looks like you've got a bit too much indenting here - tidyverse style is two spaces.